aepsych/benchmark/problem.py

```diff
@@ -233,12 +234,13 @@ def f_threshold(self, model=None) -> torch.Tensor:
             inverse_torch = model.likelihood.objective.inverse
 
             def inverse_link(x):
-                return inverse_torch(torch.tensor(x))
+                return inverse_torch(torch.tensor(x).clone().detach())
```

Comment:
Why is this cloned and detached? Also why is it converted to tensor if  we already assume that self.thresholds will be a tensor?

Reply:
You're right, there's no need to convert it again, and cloning and detaching are also unnecessary. I'll remove those, test it, and commit the changes.